resource-sharing-partners/src/test/java/test/utils/BasebibliotekGenerator.java

```diff
@@ -90,26 +81,25 @@ public static String toXml(BaseBibliotek baseBibliotek) {
         return xmlWriter.toString();
     }
 
-    private Record generateRecord(boolean shouldHaveBibNr,
-                                  boolean shouldHaveLandkode,
-                                  String specifiedNncipUri,
-                                  boolean withStengtFra,
-                                  boolean withStengtTil,
-                                  boolean withPaddr,
-                                  boolean withVaddr,
-                                  boolean withIsil,
-                                  String katsys,
-                                  List<String> eressursExcludes,
-                                  String stengt) {
+    private Record generateRecord(
+        boolean shouldHaveLandkode,
+        String specifiedNncipUri,
+        boolean withStengtFra,
+        boolean withStengtTil,
+        boolean withPaddr,
+        boolean withVaddr,
+        boolean withIsil,
+        String katsys,
+        List<String> eressursExcludes,
+        String stengt) {
         var record = new Record();
         record.setRid(incrementCurrentRidAndReturnResult());
         record.setTstamp(randomLocalDate().toString());
 
         //optional fields:
 
-        if (shouldHaveBibNr) {
-            record.setBibnr(randomString());
-        }
+        record.setBibnr(randomString());
```

Comment:
Alma code lookup krever et velkjent bibnr, og jeg har kode som tar bibnr inn i metoden. Blir litt kodekrash akkurat her.

Reply:
Alle testene mine baserer seg på at man kjenner bibNr. veldig mye var basebibliotek.getRecord().get(0).getBibNr() i testene for å hente dette ut. 
Tenkte du på at det var viktig at man vet hva bibNr er? eller tenkte du mer på at bibNr ikke kan være en randomString?

Reply:
fikset?

Reply:
bibnr brukes for å slå opp alma kode, så i min kode sender jeg alltid inn et kjent bibnr istedenfor random string. Du trenger ikke å fikse noe. Jeg kan ta merge-jobben etterpå.